lib/adama/version.rb

```diff
@@ -1,3 +1,3 @@
 module Adama
-  VERSION = "0.1.5"
+  VERSION = "0.1.6"
```

Comment:
feels like `0.2.0` would be better since Ruby 3.x.x support is big (even though the actual change is small)